core/src/main/java/org/apache/calcite/rel/logical/ToLogicalConverter.java

```diff
@@ -150,6 +151,15 @@ public ToLogicalConverter(RelBuilder relBuilder) {
       return LogicalCalc.create(visit(calc.getInput()), calc.getProgram());
     }
 
+    if (relNode instanceof TableModify) {
+      final TableModify tableModify = (TableModify) relNode;
+      final RelNode input = visit(tableModify.getInput());
+      return new LogicalTableModify(tableModify.getCluster(), tableModify.getTraitSet(),
```

Comment:
Shall we  use `LogicalTableModify#create` ?

Reply:
Thanks for review, I think `new LogicalTableModify` and `LogicalTableModify#create` are equivalent. but it seems like that we use `LogicalTableModify#create` more better.